go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/unidoc/pkcs7
+
+go 1.14
```

Comment:
does this work on earlier go versions? would be good if we can get this down to 1.11

Reply:
I tested with 1.11 version it works.
Updated to 1.11.